cvat-core/src/api-implementation.js

```diff
@@ -415,17 +415,21 @@
 
         cvat.aifred.getWorkoutMetrics.implementation = (workoutId) => aifredProxy.getWorkoutMetrics(workoutId);
 
-        cvat.aifred.getOcellusModel.implementation = (workoutId, fileName) => {
-            aifredProxy.getOcellusModel(workoutId, fileName);
-        };
+        cvat.aifred.getOcellusModel.implementation =
+            (workoutId, fileName) => aifredProxy.getOcellusModel(workoutId, fileName);
```

Comment:
Maybe this? Or is it a python convention?
```python
cvat.aifred.getOcellusModel.implementation = (workoutId, fileName) => 
    aifredProxy.getOcellusModel(workoutId, fileName);
```

Reply:
Nope Senior you've just thrown me on a mine field called "JS linter":
```
Expected no linebreak before this expression.eslintimplicit-arrow-linebreak
```
So I cannot use break line after `=>` in this project ;)

Reply:
wat!? I'd say the linter is wrong :P